File: app/src/main/java/com/song/deviceinfo/ui/soc/SOCViewModel.java
package com.song.deviceinfo.ui.soc;

import android.content.Context;

import com.song.deviceinfo.info.SOCInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/6/8.
 */
public class SOCViewModel extends NormalViewModel {

    public List<Pair<String, String>> getSOCInfo(Context context) {
        return SOCInfo.getSOCInfo(context);
    }
}

File: app/src/main/java/com/song/deviceinfo/ui/soc/SOCViewModel.kt
package com.song.deviceinfo.ui.soc

import android.content.Context
import androidx.core.util.Pair
import com.song.deviceinfo.info.SOCInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/6/8.
 */
class SOCViewModel : NormalViewModel() {
    fun getSOCInfo(context: Context?): List<Pair<String, String>> {
        return SOCInfo.getSOCInfo(context!!)
    }
}
